pages/common/composer.md

```diff
@@ -0,0 +1,31 @@
+# composer
+
+> Dependency Manager for PHP.
+
+- Interactively create a composer.json file defining the project's dependencies:
+
+`composer init`
+
+- Create a new project from a given version of an existing package:
```

Comment:
Understood the part about version, but still not clear on what does it mean by, creating a project "from" a package.